### DISCUSSION TITLE
can background job be executing while performing other tasks manually on the device locally?

### DISCUSSION THREAD

**Discussion by Antwnina** [arcus-azure/arcus.backgroundjobs#201] [Q&A] [answered]
Hi :) 
great job by the way. By initiating the background job is it possible to do some manual work locally on the same device?
Let's say i want to initiate the background job to run some flows and in parallel to do staff like writing emails etc. is this possible?

**Comment by stijnmoreels** [comment, marked as answer]
Hi! @Antwnina , thx for contacting us and for the compliment. Much appreciated. 😍
The background jobs are all behind the scenes .NET hosted services, so yes, they can be run in the background without interfering with the main application or manual work on your device.
Although, these jobs are really created with the idea to be run on a separate machine.

**Reply by Antwnina** [reply]
and this can be done only with cloud automation like in azure or aws?
is there any possibility to do it locally?

**Reply by stijnmoreels** [reply]
Yes, it can be run locally as it compiles to a regular .NET application. 👍

**Reply by Antwnina** [reply]
what's the way to do that?

**Reply by stijnmoreels** [reply]
Think there may be a misunderstanding here: the Arcus BackgroundJobs is a collection of [.NET hosted services](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/host/hosted-services?view=aspnetcore-7.0&tabs=visual-studio) that you can use as a background task for your application. It does not affect the way the application is hosted or deployed. It compiles just like any other .NET application. If you want to run something locally, you'll have to look at how to deploy your application on a custom machine, or place, but that does not have anything to do with what we provide in the Arcus BackgroundJobs repository.
Hope this helps.

**Reply by Antwnina** [reply]
Great thanks a lot for the info! <3
